packages/api/test/e2e/fees.e2e.ts

```diff
@@ -59,15 +47,29 @@ describe('fees in cennznet', () => {
             });
             const fee = await tx.fee(sender.address);
```

Comment:
Maybe this can be renamed to `calculatedFee` (I know you didn't write this originally but can fix since we're changing this already).

`gas` isn't really correct name either should be `actualFee` or something.

So then the test is `expect(calculatedFee.toString()).toEqual(actualFee.toString());`

Same with the other usages